CMakeLists.txt

```diff
@@ -232,6 +232,15 @@ add_subdirectory(${xgboost_SOURCE_DIR}/plugin)
 
 if (PLUGIN_RMM)
   find_package(rmm REQUIRED)
+
+  # Patch the rmm targets so they reference the static cudart
+  # Remove this patch once RMM stops specifying cudart requirement
+  # (since RMM is a header-only library, it should not specify cudart in its CMake config)
```

Comment:
even though it's header-only, having cudart as it's link interface in CMake target makes sense as cudart is indeed its dependency.

Reply:
The point is that the application should be able to choose between static cudart and dynamic cudart.

Reply:
@vyasr Can you comment on how RMM will handle cudart in the future?

Reply:
@trivialfis  See https://github.com/rapidsai/rmm/issues/1320

Reply:
@hcho3 @jakirkham Feel free to merge if this unblocks a critical issue with conda packaging. We can continue the discussion on the rmm thread.